### DISCUSSION TITLE
redirect dns traffic

### DISCUSSION THREAD

**Discussion by krieg033** [TechnitiumSoftware/DnsServer#280] [Q&A]
How to redirect for example: requirement google.com And redirect mywebsite or my local apache (localhost)

**Comment by ShreyasZare** [comment]
You can just create a primary zone for the domain you require and then add an A record with the IP address of your web server. One the web server, you will have to configure virtual host so that the domain is identified by the web server and the relevant pages are shown. Otherwise, the default website on your web server will be displayed.

If you wish to only redirect for a particular domain or its subdomain such that rest all other records should resolve normally then create a conditional forwarder zone for the domain instead. Use "this-server" as while creating the zone as the forwarder so that all requests are forwarded internally while you can add A record or any other record which overrides the domain's existing record.

**Reply by XZNT** [reply]
I have a short question, can i make a custom dns record and point it to my local web server (apache) that i made publicly visible, how can my domain be accessed? I want to turn it to public is it possible?

**Reply by ShreyasZare** [reply]
You can host your entire zone or just a subdomain zone on Technitium DNS server and host it publicly such that anyone on the Internet will be able to resolve it. I am running my domain `technitium.com` on Technitium DNS server itself.

You can run the DNS server on a cloud server or locally on your network if your ISP provides a static IP address. Read [this blog post](https://blog.technitium.com/2022/06/how-to-self-host-your-own-domain-name.html) to know how to host your own domain name to allow public resolution.
